### DISCUSSION TITLE
GitHub pages won't show a new post

### DISCUSSION THREAD

**Discussion by emagar** [community/community#73332] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,

I published a new GitHub post following the same steps I've used for years. I successfully pushed the post to my repository [here](https://github.com/emagar/emagar.github.io/blob/ab2543f87886ec2654afb504b20dbf9172e1fa44/_posts/2023-10-19_quisiera_ser_un_pes.html); however, the new post won't appear on the site. What is the problem?

Here is my site: https://emagar.github.io/

**Comment by eminemahjoub** [comment]
It seems like you're experiencing an issue with your GitHub Pages website not reflecting the changes you've made. There could be several reasons for this, and I'll outline some common troubleshooting steps to help you identify and resolve the problem:

Check GitHub Pages Settings:

Make sure your repository is correctly configured for GitHub Pages. Go to your repository's settings, and in the "GitHub Pages" section, verify that you have selected the correct branch (usually main or master) as the source for your GitHub Pages site.
Commit and Push:

Ensure that you have committed and pushed your changes to the GitHub repository. Without these actions, the changes won't be reflected on your site.
Branch Name:

Double-check that you are pushing changes to the correct branch. If you are using the gh-pages branch or another custom branch for GitHub Pages, make sure you push your changes to that branch.
Jekyll Build (If Using Jekyll):

If your GitHub Pages site is built using Jekyll, make sure you've built the site locally and pushed the generated _site or docs folder to your repository.
GitHub Pages Build Time:

Sometimes, GitHub Pages can take a few minutes to build your site after you push changes. Be patient and check your site again after a little while.
Caching Issues:

Your browser may be caching the old version of your site. Try opening your site in an incognito or private browsing window or clear your browser cache.
Custom Domain:

If you're using a custom domain with GitHub Pages, ensure that your DNS settings are correctly configured.
GitHub Actions or CI/CD Pipelines:

If you're using GitHub Actions or a CI/CD pipeline to build and deploy your site, check if there are any issues with your workflow configuration.
Repository Permissions:

Ensure that you have the necessary permissions to push changes to the repository. If it's a private repository, check that collaborators have access.
404 Page:

If you're getting a 404 error, make sure there's an index.html or README.md file in your repository's root.
Error Messages:

Look for error messages or build failures in your GitHub repository. GitHub will provide error details that can help you pinpoint the issue.
Contact GitHub Support:

If none of the above steps work, and you're sure everything is set up correctly, consider reaching out to GitHub Support for further assistance.

**Comment by emagar** [comment, marked as answer]
Thank you @eminemahjoub for your steps. None worked. The problem had to do with the name I gave to the post: I removed underscores (_) from my file name, and it now appears in my web page's index file. I don't remember reading that underscores would raise problems in GitHub pages, but apparently that is the case.

**Reply by chezydan** [reply]
same here! Happy to find this one. Your comment saved me!
